src/JustSaying/Extensions/ChunkExtensions.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Diagnostics;
+
+namespace JustSaying.Extensions;
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
Presumably we only need this if targeting below .NET 6 so this could be wrapped in a hash-define?

Reply:
Perhaps we should start to include `net6.0` in the built TFMs, in which case @hwoodiwiss suggested we conditionally include this file for non-net6 tfms.

Reply:
I think that's already been done.

Reply:
Ah I see, the `#if !NET6_0_OR_GREATER` thing, yeah this will always evaluate to `true`, I'm suggesting adding `net6.0` as a TFM, it increases the package size a little, but we can start to use the newer stuff when it's available.